### PR TITLE
always use the same API keys for demo

### PR DESCRIPTION
**Pull request by coolo**
Makes it easier to keep client.conf around in development

### CODE REVIEW

**Comment by aaannz** [comment]
Can you create keys with some short expiration (e.g. 1 day) there (and subsequent updates for it). Or at least some remove-before-launch warning in doc.

I.e. admin, after testing and setting everything up, decides his instance is ready for production, change auth model, open firewall, but leave original DB. This ApiKey can then be misused by 3rd party.

**Comment by coolo** [comment]
expiry sounds fine

**Comment by coolo** [comment]
redone according to your suggestion

File: lib/OpenQA/Auth/Fake.pm [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 package OpenQA::Auth::Fake;
 use OpenQA::Schema::Result::Users;
+use DBIx::Class::Timestamps qw/now/;
```
Comment (aaannz) [inline lib/OpenQA/Auth/Fake.pm]: Now I don't understand why importing 'now'

Comment (coolo) [inline lib/OpenQA/Auth/Fake.pm]: I tried now first, but couldn't add to it, so went with from_epoch :)

**Comment by coolo** [comment]
removed again

**Pull request merged**